### PR TITLE
Add SKYSAKURAH743 board to the wiki

### PR DESCRIPTION
**Pull request by BloodSakura3774**

### CODE REVIEW

**Comment by haslinghuis** [comment]
@VitroidFPV pls check what is wrong here ?

**Comment by VitroidFPV** [comment]
Will investigate when I'm home 🔎

**Comment by VitroidFPV** [comment]
I see, it's some odd automated linting behavior. Will look closer

**Comment by VitroidFPV** [comment]
Progress. The template file will also need to be updated to match, now just to fix the image embeds

**Comment by VitroidFPV** [comment]
All looks good now - should be ready for review

**Comment by nerdCopter** [comment]
image

**Comment by VitroidFPV** [comment]
Odd that comments make it through, I'll see what can be done about that.

File: docs/wiki/boards/current/SAKURAH743/SAKURAH743.mdx [hunk 1]
```diff
@@ -237,11 +237,11 @@ import SpecGrid from '@site/src/components/SpecGrid'
 <TabItem value="photos" label="Photos">
 ![TOP](./SAKURAH743-images/SkySakuraH743_top.png)
 ![BOTTOM](./SAKURAH743-images/SkySakuraH743_bottom.png)
-![TOPpit](./SAKURAH743-images/SkySakuraH743_TOP.jpg)
-![BOTTOMpitwithpower](./SAKURAH743-images/SkySakuraH743_bot_P.jpg)
-![BOTTOMpit](./SAKURAH743-images/SkySakuraH743_bot.jpg)
-![POWERtop](./SAKURAH743-images/SkySakuraH743_powerB.jpg)
-![POWERbot](./SAKURAH743-images/SkySakuraH743_powerT.jpg)
+![TOPpit](./SAKURAH743-images/SkySakuraH743_TOP.JPG)
```
Comment (haslinghuis) [inline docs/wiki/boards/current/SAKURAH743/SAKURAH743.mdx:240]: :thinking: File extensions should be lowercase, rather change the source.

Comment (VitroidFPV) [reply docs/wiki/boards/current/SAKURAH743/SAKURAH743.mdx:240]: Changing the image filetype to lowercase wasn't detected as any change, it wouldn't let me commit an empty commit. Unsure what's going on there

**Review by haslinghuis — APPROVED** [review]
接線圖是中文的

**Review by nerdCopter — APPROVED** [review]

**Comment by BloodSakura3774** [comment]
Wiring diagram has both English and Chinese, this is what we include in the box

**Pull request merged**